sdk/appconfiguration/app-configuration/samples/helloworldWithLabels.ts

```diff
@@ -1,6 +1,11 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+// This sample builds on helloworld.ts and shows you how to use labels.
+// Labels allow you to add an extra dimension for your setting and gives
+// you a simple way to create conventions like production vs beta settings,
+// development vs testing, etc...
```

Comment:
`etc` and `...` together feel excessive and the tone is a little weird.

Maybe `a simple way to create conventions like production vs beta or development environments.`?

Reply:
Agreed. I've got an update coming and I'll also link to the product documentation for labels which explains things even further.